readme.md

```diff
@@ -132,10 +132,26 @@ Some resources allow the results to be filtered. This will be documented with th
 * ```~``` Contains
 * ```~~``` Wildcard search (%)
 
-To filter a resource, add a filter string to the 'q' parameter. To apply multiple filters, send multiple filter strings as an array with the `q[]` parameter.
+
+**Fitler String Format**
```

Comment:
echo?

This is one of my most common typos. Happens to me all the time in code.